spec/spec_helper.rb

```diff
@@ -8,9 +8,13 @@
 require "bundler"
 Bundler.setup
 
-ENV['RAILS'] = '4.1.9'
-
 ENV['RAILS_ENV'] = 'test'
+# Ensure the Active Admin load path is happy
+require 'rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.